src/zepto.js

```diff
@@ -51,9 +51,9 @@ var Zepto = (function() {
 
   zepto.matches = function(element, selector) {
     if (!selector || !element || element.nodeType !== 1) return false
-    var matchesSelector = element.webkitMatchesSelector || element.mozMatchesSelector ||
+    var matches = element.matches || element.webkitMatchesSelector || element.mozMatchesSelector ||
```

Comment:
:)Thanks for your reminder.
What about puting msMatchesSelector after webkitMatchesSelector  duo to the range of  useage for less judging.
And what about deleting oMatchesSelector ? Because Opera support matches  and support querySelectorAll fully after version 27. There is no such oMatchesSelector defined in Opera.
Ref http://caniuse.com/#search=matches  Ref http://caniuse.com/#search=queryselectorall.It means that zepto.matches just support Opera27+.